### ISSUE TITLE
ctest error

### ISSUE THREAD

**Issue by alexhxia** [key4hep/k4MarlinWrapper#76]
Hello,
When I run : `ctest --verbose -R simple_processors` (with `source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh`)
I have this output (for simple_processors) :
```
1: AlgorithmManager    ERROR Algorithm of type LcioEvent is unknown (No factory available).
1: AlgorithmManager    ERROR Unknown error -1342251347
1: AlgorithmManager    ERROR More information may be available by setting the global jobOpt "PluginDebugLevel" to 1
1: EventLoopMgr        ERROR Unable to create Top Algorithm LcioEvent with name LcioEvent
1: EventLoopMgr        ERROR Failed to initialize Top Algorithms streams.
1: EventLoopMgr        DEBUG Error Initializing base class MinimalEventLoopMgr.
1: ServiceManager      ERROR Unable to initialize Service: EventLoopMgr
1: ApplicationMgr      ERROR Application Manager Terminated with error code 1
```
I don't know how solve that ?

**Comment by vvolkl** [comment]
Hi, this will be fixed by #73 as well.

**Comment by alexhxia** [comment]
```
$ ctest
Test project /home/ilc/ahocine/k4MarlinWrapper/build
      Start  1: simple_processors
 1/10 Test  #1: simple_processors ................   Passed    4.38 sec
      Start  2: simple_processors2
 2/10 Test  #2: simple_processors2 ...............   Passed    4.66 sec
      Start  3: clicRec
 3/10 Test  #3: clicRec ..........................   Passed   83.19 sec
      Start  4: converter_constants
 4/10 Test  #4: converter_constants ..............   Passed    5.58 sec
      Start  5: edm_converters
 5/10 Test  #5: edm_converters ...................   Passed    1.60 sec
      Start  6: all_events_bounds
 6/10 Test  #6: all_events_bounds ................   Passed    4.33 sec
      Start  7: over_total_events
 7/10 Test  #7: over_total_events ................   Passed    4.55 sec
      Start  8: same_num_io
 8/10 Test  #8: same_num_io ......................   Passed    2.33 sec
      Start  9: clicRec_edm4hep_input
 9/10 Test  #9: clicRec_edm4hep_input ............   Passed   98.87 sec
      Start 10: clicRec_lcio_mt
10/10 Test #10: clicRec_lcio_mt ..................   Passed   91.77 sec

100% tests passed, 0 tests failed out of 10

Total Test time (real) = 301.30 sec
```
Thank you, it's working.
